runtime/gc_base/RootScanner.cpp

```diff
@@ -1044,6 +1050,12 @@ MM_RootScanner::scanClearable(MM_EnvironmentBase *env)
 	scanOwnableSynchronizerObjects(env);
 	scanContinuationObjects(env);
 
+	J9JavaVM *vm = (J9JavaVM*)env->getOmrVM()->_language_vm;
+	J9JITConfig *jitConfig = vm->jitConfig;
+	if (NULL != jitConfig->methodsToDelete) {
```

Comment:
could jitConfig be null (if running -Xint)?

Reply:
update it to 
`if ((NULL != jitConfig) && (NULL != jitConfig->methodsToDelete)) {
`